.github/workflows/node.js.yml

```diff
@@ -0,0 +1,28 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-and-lint:
+    name: Build and lint
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 18
+        cache: yarn
+    - name: Install dependencies
+      run: yarn --frozen-lockfile
+    - name: Build application
+      run: yarn build
+      env: 
+        CI: false
```

Comment:
How come this is needed? We're in the pipeline space now, so I would have expected this to be true.

Do you have a link for the documentation as well, please? I cannot find it online.

Reply:
When I ran without this, it was failing on warnings.  I'm not sure on documentation, in this case I just searched for people seeing similar behaviour and this was the suggested fix (which worked!).